app/helpers/application_helper.rb

```diff
@@ -22,4 +22,11 @@ def render_flash_message(flash_value)
     end
   end
 
+  def user_has_membership_application?
+    current_user.try(:membership_applications).try(:any?) && current_user.try(:membership_applications).try(:last).try(:persisted?)
+  end
+
+  def member_has_company?
+    current_user.is_member && !current_user.membership_applications.last.company.nil?
+  end
```

Comment:
1) If user does not have a membership application, this will result in an exception being raised since `company` method will be called on `nil`.

2) See comment #2 above.